pkgs/by-name/fl/fleet/package.nix

```
@@ -0,0 +1,46 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  buildGoModule,
+  writableTmpDirAsHomeHook,
+  versionCheckHook,
+}:
+
+buildGoModule rec {
+  pname = "fleet";
+  version = "4.67.1";
+
+  src = fetchFromGitHub {
+    owner = "fleetdm";
+    repo = "fleet";
+    tag = "fleet-v${version}";
+    hash = "sha256-cZ0YTFcyPt7NMZUDZCdlVPTuhwRy7mTp7JCdINqiwOM=";
+  };
+  vendorHash = "sha256-gFAotYho18Jn8MaFK6ShoMA1VLXVENcrASvHWZGFOFg=";
+
+  subPackages = [
+    "cmd/fleet"
+  ];
+
+  ldflags = [
+    "-X github.com/fleetdm/fleet/v4/server/version.appName=${pname}"
```

referencing pname is bad practice  
```suggestion
    "-X github.com/fleetdm/fleet/v4/server/version.appName=fleet"
```